file/readfile.go

```diff
@@ -91,13 +91,19 @@ func readContent(reader io.Reader) (*Content, error) {
 	if err != nil {
 		return nil, errors.Wrap(err, "validating file content")
 	}
-	err = yaml.Unmarshal(bytes, &content)
+	err = yamlUnmarshal(bytes, &content)
 	if err != nil {
 		return nil, err
 	}
 	return &content, nil
 }
 
+// yamlUnmarshal is a wrapper around yaml.Unmarshal to ensure that the right
+// yaml package is in use. The verification for this is done using a test.
+func yamlUnmarshal(bytes []byte, v interface{}) error {
+	return yaml.Unmarshal(bytes, v)
```

Comment:
optional suggestion: import by a nonstardard name, e.g. `ghodssyaml`, to make it harder for goimports to tamper with this import